doc/source/how-to/index.rst

```diff
@@ -6,8 +6,103 @@ effective and efficient Python libraries to interface with Ansys products and
 services. Topics also demonstrate how apps and complex services expose
 functionalities such as logging, data transfer, and app APIs.
 
+.. grid:: 3
+
+    .. grid-item-card:: :fas:`fa-regular fa-screwdriver-wrench` Environment setup
+      :link: setting-up
+      :link-type: doc
+      :padding: 2 2 2 2
+
+      How to set up a development environment.
+
+    .. grid-item-card:: :fas:`fa-solid fa-code-compare` Versions
+      :link: supporting-python-versions
+      :link-type: doc
+      :padding: 2 2 2 2
+
+      Supporting Python versions.
```

Comment:
Write as a how-to description? (All those that I suggest may need to be tweaked for technical accuracy. I'd just love to see consistency in the descriptions on each card.)

How to know which Python versions to support.